### ISSUE TITLE
Bug: psycopg2.errors.NotNullViolation: null value in column "base_shipping_price_amount" violates not-null constraint

### ISSUE THREAD

**Issue by DustinBracy** [saleor/saleor#11225]
### What are you trying to achieve?

During the 2nd checkoutComplete mutation after receiving a successful payment response from Stripe, using the builtin Stripe plugin, I'm receiving an unhandled GraphQL error: 
`Message: null value in column "base_shipping_price_amount" violates not-null constraint
DETAIL:  Failing row contains (2022-11-16 22:58:00.982986+00, 3de319fe-d62f-5147-96d1-4366b80fccd0, bracy.dustin@gmail.com, 16b233cc-4a5c-4a6b-8b09-7d989e131278, 82, null, null, 4.990, null, en, 0.000, 5.420, 0.000, unfulfilled, null, null, t, , 0, 96f98752-5978-43b7-8c48-42fa133b449b, USD, {}, {}, 1, null, 0.0870, 5.420, 4.990, 0.000, checkout, null, null, , 2022-11-16 22:58:00.983332+00, f, 41, null, 0.000, none, none, null, f, f, null).
, Location: [{"line":2,"column":3}], Path: checkoutComplete`

### Steps to reproduce the problem

1. Create checkout with item of any price, with no shipping required, assign billing address + email
2. execute checkoutPaymentCreate with checkoutId + amount
3. execute checkoutComplete with checkout id + paymentCreate response
4. execute Stripe confirmPayment method (I'm using stripe's react-native useConfirmPayment hook)
5. obtain id + paymentData from Stripe successful payment response, send to checkoutComplete mutation 
6. receive error

### What did you expect to happen?

I expected the 2nd checkoutComplete process to complete successfully and issue me an order.  This was working in prior versions, and I have tried downgrading to the latest 3.4 saleor release and still get the same error.  I ran migrations successfully, and running them again gives me a 'no migrations to apply' message.

### Logs

2022-11-16 22:58:00,986 ERROR saleor.graphql.errors.unhandled A query failed unexpectedly [PID:26:ThreadPoolExecutor-0_0]
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.NotNullViolation: null value in column "base_shipping_price_amount" violates not-null constraint
DETAIL:  Failing row contains (2022-11-16 22:58:00.982986+00, 3de319fe-d62f-5147-96d1-4366b80fccd0, bracy.dustin@gmail.com, 16b233cc-4a5c-4a6b-8b09-7d989e131278, 82, null, null, 4.990, null, en, 0.000, 5.420, 0.000, unfulfilled, null, null, t, , 0, 96f98752-5978-43b7-8c48-42fa133b449b, USD, {}, {}, 1, null, 0.0870, 5.420, 4.990, 0.000, checkout, null, null, , 2022-11-16 22:58:00.983332+00, f, 41, null, 0.000, none, none, null, f, f, null).

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/graphql/execution/executor.py", line 452, in resolve_or_error
    return executor.execute(resolve_fn, source, info, **args)
  File "/usr/local/lib/python3.9/site-packages/graphql/execution/executors/sync.py", line 16, in execute
    return fn(*args, **kwargs)
  File "/app/saleor/graphql/core/mutations.py", line 379, in mutate
    response = cls.perform_mutation(root, info, **data)
  File "/app/saleor/graphql/checkout/mutations/checkout_complete.py", line 250, in perform_mutation
    order, action_required, action_data = complete_checkout(
  File "/app/saleor/checkout/complete_checkout.py", line 1277, in complete_checkout
    order, action_required, action_data = complete_checkout_post_payment_part(
  File "/app/saleor/checkout/complete_checkout.py", line 813, in complete_checkout_post_payment_part
    order = _create_order(
  File "/usr/local/lib/python3.9/contextlib.py", line 79, in inner
    return func(*args, **kwds)
  File "/app/saleor/checkout/complete_checkout.py", line 505, in _create_order
    order = Order.objects.create(
  File "/usr/local/lib/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/query.py", line 453, in create
    obj.save(force_insert=True, using=self.db)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 739, in save
    self.save_base(using=using, force_insert=force_insert,
  File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 776, in save_base
    updated = self._save_table(
  File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 881, in _save_table
    results = self._do_insert(cls._base_manager, using, fields, returning_fields, raw)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/base.py", line 919, in _do_insert
    return manager._insert(
  File "/usr/local/lib/python3.9/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/query.py", line 1270, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "/usr/local/lib/python3.9/site-packages/django/db/models/sql/compiler.py", line 1416, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 98, in execute
    return super().execute(sql, params)
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 66, in execute
    return self._execute_with_wrappers(sql, params, many=False, executor=self._execute)
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 75, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "/app/saleor/graphql/views.py", line 47, in tracing_wrapper
    return execute(sql, params, many, context)
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
  File "/usr/local/lib/python3.9/site-packages/django/db/utils.py", line 90, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.9/site-packages/django/db/backends/utils.py", line 84, in _execute
    return self.cursor.execute(sql, params)
django.db.utils.IntegrityError: null value in column "base_shipping_price_amount" violates not-null constraint
DETAIL:  Failing row contains (2022-11-16 22:58:00.982986+00, 3de319fe-d62f-5147-96d1-4366b80fccd0, bracy.dustin@gmail.com, 16b233cc-4a5c-4a6b-8b09-7d989e131278, 82, null, null, 4.990, null, en, 0.000, 5.420, 0.000, unfulfilled, null, null, t, , 0, 96f98752-5978-43b7-8c48-42fa133b449b, USD, {}, {}, 1, null, 0.0870, 5.420, 4.990, 0.000, checkout, null, null, , 2022-11-16 22:58:00.983332+00, f, 41, null, 0.000, none, none, null, f, f, null).

### Environment

Saleor version: … 3.7.35
OS and version: … docker image python:3.9

**Comment by DustinBracy** [comment]
Just an update on this.  I can recreate this error much more simply: by trying to create an order from the dashboard manually.  Click Orders > Orders > Create Order.  Select a channel (I have only one), click confirm, and receive the error message.  This continues to happen in saleor 3.7.37 with dashboard 3.8.5.

**Comment by peelar** [comment]
Hello @DustinBracy 👋 

Is your code modified in any way or is it simply a clone of Saleor?

**Comment by DustinBracy** [comment]
It is a clone of Saleor.  I was able to resolve it by updating to Saleor 3.8.8 (with 3.8.5 dashboard) and deleting all data in the orders, checkout, payments, warehouse reservations and related tables directly in pgadmin.  Obviously, this would not be possible in production, but I am still a few weeks away from going live.  I created a new docker compose stack and populated test data using manage.py populatedb and was not able to recreate the error.  I wonder if something got corrupted in the database (I started using Saleor on 3.0 when it was still in early beta).   We can close this as I can no longer reproduce the error.
